wcag-ruleset-review-process.md

```diff
@@ -1,77 +1,89 @@
+### 1. Draft proposal
 
-### Submitting
+New rules, and changes to rules are made through GitHub pull requests. Any member of either ACT group can author such a pull request. A pull request must have approval from three independent reviewers before it can proceed to step 2. These reviewers can not work for the same organization, and must not have been involved in writing the pull request. 
 
-The rule provider creates an issue in the [ACT GitHub repository](https://github.com/w3c/wcag-act/issues). The issue must include the following information:
+The pull request description must include information on the type of change the pull request makes, and what happens in the "call for review" phase. There are three options:
 
-1. URL of the rule
-2. License of the rule
-3. Information about any [implementations](#implementations)
+1. New rules, and significant changes that impact multiple rules should have a **two week** call for review. 
+2. Updates to rules, and changes to definitions used in a small number of rules have a **one week** call for review.
+3. Editorial changes, including minor changes to test cases can **skip** call for review.
 
-### Surveying
+### 2. Call for review
 
-A facilitator of the ACT Task Force creates a survey for the rule, in which Task Force members are asked the following questions:
+Once a pull request has the necessary approvals, the author of the pull request sends an e-mail to the ACT Rules community group mailing list with the text "[for review]" in the subject. The e-mail must include a link to the pull request, and an indication of the duration and end date of the call for review. 
+
+1. If no changes are requested on the pull request during the review period, the author is free to merge the pull request. 
+2. Otherwise, the author must resolve the requested changes. A comment is resolved when it has approval from the original commenter, and at least two other reviewers. This can happen during or after the call for review. Depending on the change, one of the following happens:
+  1. If the changes are editorial, the pull request can be merged any time after the review period.
+  2. If the changes are not editorial, repeat step 1, followed by a **one week** call for review.
+
+When a pull request is merged, the rule is published on the WAI website with a clear indicator that the rule, or the update to the rule is a "proposal". Additionally, metadata for the rule is published, so that implementors of ACT rules can create an implementation report for the proposed rule.
+
+### 3. Call for implementation
+
+When there is at least **one complete implementation** of the rule, the proposal can proceed to the validation phase. See [complete implementations](#complete-implementations).
```

Comment:
Back in https://github.com/act-rules/act-rules.github.io/issues/577 and https://www.w3.org/2019/06/27-act-r-minutes.html#item03 the CG decided that 3 implementations were needed. I couldn't find when we decided to go down to only 1 implementation.
Apparently, we also decided for a CG CFC before sending to the TF, which never happened… So I guess this was maybe changed at a later point, but when?

Reply:
The ACT TF added its own requirements for implementations in August 2020. Here's the PR of it: #472. With this proposal the CG now adopts that requirement.